server/series/utils.py

```diff
@@ -0,0 +1,112 @@
+from django.db.models import Q, QuerySet, Value
+from django.db.models.functions import Concat
+
+from server.core.models import Player, Team
+from server.types import message_response
+
+from .models import Series, SeriesRegistration
+
+
+class RegistrationError(Exception):
+    def __init__(self, message: str) -> None:
+        self.message = message
+
+    def __str__(self) -> str:
+        return self.message
+
+
+def can_invite_player_to_series_roster(
```

Comment:
also need to check if the teams have atleast 1 spot with current number of invitations+registrations count and `series_roster_max_players`

Reply:
Ah forgot the roster size check. Do you think we should do this check while accepting the invitation ?

Reply:
so we should check both send and accept i feel.. Basically get a unilque set of player ids invitation is sent(which is not expired) and number of registrations

Reply:
ok, so the roster limit check alone we do for invitations ? we discussed that we aren't going to do any checks while sending out invitations right ?

Reply:
done